components/devtools/actors/inspector/highlighter.rs

```diff
@@ -21,6 +24,8 @@ pub struct HighlighterMsg {
 
 pub struct HighlighterActor {
     pub name: String,
+    pub script_channel: IpcSender<DevtoolScriptControlMsg>,
```

Comment:
I would go with the `_sender` naming here that we are using in other parts of the code as "channel" is a bit ambiguous.